src/components/Swap/Output.tsx

```diff
@@ -74,37 +67,37 @@ export default function Output({ disabled, focused, children }: PropsWithChildre
   return (
     <DynamicThemeProvider color={color}>
       <OutputColumn hasColor={hasColor} gap={0.5}>
-        <Row>
-          <ThemedText.Subhead1 color="secondary">
-            <Trans>For</Trans>
-          </ThemedText.Subhead1>
-        </Row>
-        <TokenInput
-          amount={amount}
-          currency={swapOutputCurrency}
-          disabled={disabled}
-          field={Field.OUTPUT}
-          onChangeInput={updateSwapOutputAmount}
-          onChangeCurrency={updateSwapOutputCurrency}
-          loading={isLoading}
-        >
-          <ThemedText.Body2 color="secondary" userSelect>
-            <Row>
-              <USDC gap={0.5} isLoading={isRouteLoading}>
-                {outputUSDC && `$${formatCurrencyAmount(outputUSDC, 6, 'en', 2)} `}
-                {impact && <ThemedText.Body2 color={impact.warning}>({impact.toString()})</ThemedText.Body2>}
-              </USDC>
-              {balance && (
-                <Balance color={focused ? 'secondary' : 'hint'}>
-                  Balance: <span>{formatCurrencyAmount(balance, 4, i18n.locale)}</span>
-                </Balance>
-              )}
-            </Row>
-          </ThemedText.Body2>
-        </TokenInput>
-        <Rule />
-        {children}
-        {disableBranding ? <Footer /> : <BrandedFooter />}
+        <MarginWrapper>
+          <Row>
+            <ThemedText.Subhead1 color="secondary">
+              <Trans>For</Trans>
+            </ThemedText.Subhead1>
+          </Row>
+          <TokenInput
+            amount={amount}
+            currency={swapOutputCurrency}
+            disabled={disabled}
+            field={Field.OUTPUT}
+            onChangeInput={updateSwapOutputAmount}
+            onChangeCurrency={updateSwapOutputCurrency}
+            loading={isLoading}
+          >
+            <ThemedText.Body2 color="secondary" userSelect>
+              <Row>
+                <USDC gap={0.5} isLoading={isRouteLoading}>
+                  {outputUSDC && `$${formatCurrencyAmount(outputUSDC, 6, 'en', 2)} `}
+                  {impact && <ThemedText.Body2 color={impact.warning}>({impact.toString()})</ThemedText.Body2>}
+                </USDC>
+                {balance && (
+                  <Balance color={focused ? 'secondary' : 'hint'}>
+                    Balance: <span>{formatCurrencyAmount(balance, 4, i18n.locale)}</span>
+                  </Balance>
+                )}
+              </Row>
+            </ThemedText.Body2>
+          </TokenInput>
+        </MarginWrapper>
+        <Toolbar />
```

Comment:
I prefer to keep this as a separate element in Swap/index.tsx, so that Input.tsx can more closely mirror Output.tsx. Is there a reason to prefer it as part of Output.tsx?

Reply:
Added Toolbar here to make it easy to accomplish the background color/border-radius/border-color arrangement necessary here:
![image](https://user-images.githubusercontent.com/5773490/194934947-b1a607d8-c662-483f-bcda-e5a69f4b084e.png)

It's not technically required, but the diff is much smaller if we do it this way.

Reply:
(No longer doing it this way. Bit the bullet and put it up a level at `Swap`.)